cropper/app/model/ExportRequest.scala

```diff
@@ -38,7 +39,13 @@ object ExportRequest {
   def boundsFill(dimensions: Dimensions): Bounds = Bounds(0, 0, dimensions.width, dimensions.height)
 
   def toCropSpec(cropRequest: ExportRequest, dimensions: Dimensions): CropSpec = cropRequest match {
-    case FullExportRequest(uri)          => CropSpec(uri, boundsFill(dimensions), None, FullExport)
+    case FullExportRequest(uri)          =>
+      CropSpec(
+        uri,
+        boundsFill(dimensions),
+        AspectRatio.calculate(dimensions.width, dimensions.height).flatMap(_.friendly),
```

Comment:
Thought about making calculate take Dimensions ( which are just width: Int and height: Int ) but it seemed like too much. maybe i just need to embrace typing ^_^